src/nyc_trees/apps/mail/views.py

```diff
@@ -56,6 +58,16 @@ def notify_group_mapping_approved(request, group, username):
     user = get_object_or_404(User, username=username)
     reservations_url = request.build_absolute_uri(reverse('reservations'))
     return _send_to(user,
-                    GROUP_MAPPING_APPROVED,
+                    _GROUP_MAPPING_APPROVED,
                     group=group,
                     reservations_url=reservations_url)
+
+
+def send_reservation_reminder(user_id, **kwargs):
+    user = get_object_or_404(User, id=user_id)
+    reservations_url = ('http://%s/%s/' %
```

Comment:
I can't thing of a way to avoid hard-coding the protocol, so I would prefer we use "https://" which is more correct, and works everywhere except dev, where we understand how to deal with it.